roles/confd/templates/haproxy.tmpl.j2

```diff
@@ -55,6 +55,9 @@ listen master_direct
     maxconn {{ haproxy_maxconn.master }}
     option tcplog
     option httpchk OPTIONS /primary
+{% if haproxy_log_format is defined %}
```

Comment:
@jimnydev let's duplicate to the haproxy conf [file](https://github.com/vitabaks/postgresql_cluster/blob/master/roles/haproxy/templates/haproxy.cfg.j2) in case someone doesn't want to use confd service.

Reply:
Of course, updated. Thanks!